webpack.config.js

```diff
@@ -221,6 +221,105 @@ module.exports = (env, argv) => {
 
                 // :TCHAP:
                 "~tchap-web": path.resolve(__dirname, "."),
+                // Hack in order to include the fonts declare in tchap compound-design-tokens
```

Comment:
I added this css for the marianne css, will it help you? 

https://github.com/tchapgouv/compound-design-tokens/blob/68c9c2ff91381836e0ce6a78b29f73b84a6e404c/source_assets/fonts/web/Marianne.css

Reply:
yes it is taken from here. But the hack below is because I am indeed using this font, but for an unknown reason he doesnt seems to find the correct package to the files. That is why i created those aliases. Will need to remove the day we find the pb :shrug: